Automattic/studio
# Disable import/export for sites not using SQLite integration

Related to https://github.com/Automattic/dotcom-forge/issues/9395

## Proposed Changes

- Add a function to determine if SQLite integration is installed on a site.
- Add an IPC handler that determines if the Import/Export feature is supported for a site. For now, the condition we check is if SQLite integration is installed for that site as the import/export commands used only support SQLite databases. In the future, we could add more.
- Render a warning notice in the Import/Export tab if the site doesn't have the SQLite integration.

<img width="922" alt="Captura de pantalla 2024-09-26 a las 14 13 47" src="https://github.com/user-attachments/assets/2b0253db-8b2d-4274-9ccc-2002727c3918">


## Testing Instructions



- Create a site.
- Navigate to the site folder.
- Remove the file `wp-content/db.php`.
- Navigate to the Import/Export tab.
- Observe a warning notice is displayed and that the Import/Export functionality is disabled.
- Select another site.
- Observe the Import/Export functionality is enabled and working.

## Pre-merge Checklist
